auth-web/src/models/Organization.ts

```diff
@@ -19,6 +19,8 @@ export interface Organization {
   orgType?: string;
   members?: Member[];
   invitations?: Invitation[];
+  // eslint-disable-next-line camelcase
+  access_type: string;
```

Comment:
Maybe this is already in progress but can we convert this to camelCase in the back-end like with the other fields?

Reply:
Done